WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenu.swift

```diff
@@ -15,19 +15,6 @@ struct InPersonPaymentsMenu: View {
                                 .ignoresSafeArea()
                         }
 
-                    ScrollViewSection {
-                        PaymentsRow(image: Image(uiImage: .moneyIcon),
-                                    title: Localization.collectPayment)
-                        .accessibilityAddTraits(.isButton)
-                        .accessibilityIdentifier(AccessibilityIdentifiers.collectPaymentRow)
-                        .onTapGesture {
-                            viewModel.collectPaymentTapped()
-                        }
-                    } header: {
-                        Text(Localization.paymentActionsSectionTitle.uppercased())
```

Comment:
nit: `Localization.paymentActionsSectionTitle` could probably be deleted as well.

Reply:
Good catch! Done in 9c328e8c772bdc89cc31baea3d61e7fcb29d3bf5